site/src/components/data/coinbase.md

```diff
@@ -0,0 +1,28 @@
+---
+layout: component
+title: Coinbase
+component: data/feed/coinbase.js
+namespace: data
+
+example-code: |
+  var feed = fc.data.feed.coinbase();
+
+  // fetch some data!
+  feed(function(error, data) {
+    d3.select("#coinbase")
+      .text(JSON.stringify(error ? error : data, null, 2));
+  });
+---
+
+Fetch data from the [Coinbase exchange API](https://docs.exchange.coinbase.com/#market-data). The `product`, `start`, `end` and `granularity` can be customised, for more information on the parameters see the Coinbase docs.
```

Comment:
Might be worth noting that the component expects `start` and `end` parameters as `Date` instances, rather than the ISO-date strings that the coinbase API expected.